src/main/java/com/poiji/bind/mapping/SheetColumns.java

```diff
@@ -0,0 +1,68 @@
+package com.poiji.bind.mapping;
+
+import com.poiji.exception.PoijiException;
+import com.poiji.option.PoijiOptions;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SheetColumns {
+
+    public static Sheet sheetAdaptor(Sheet sheet, PoijiOptions options) {
+
+        try {
+            Workbook workbook = sheet.getWorkbook();
+            FormulaEvaluator evaluator = workbook.getCreationHelper().createFormulaEvaluator();
+            evaluator.setIgnoreMissingWorkbooks(true);
+            Workbook result = new XSSFWorkbook();
+            Sheet resultSheet = result.createSheet();
```

Comment:
creating a new sheet from scratch hmm...

Reply:
I guess better create new sheet and transfer required info to new sheet from huge old sheet.